src/components/Menu/components/forms/NewCategoryForm.js

```diff
@@ -1,18 +1,14 @@
 import { FormControl, Input } from "@chakra-ui/react";
-import useField from "hooks/useField";
+import { useForm } from "react-hook-form";
 
 const NewCategoryForm = ({ initialName }) => {
-  const categoryName = useField("text", initialName);
+  const { register } = useForm({
+    defaultValues: { name: initialName },
```

Comment:
Rather than doing this way, we can create a property "defaultValues" and pass it to useForm like this: 

```js
const NewCategoryForm = ({ defaultValues }) => {
  const { register } = useForm({
    defaultValues
   })
```